src/index.ts

```diff
@@ -215,13 +217,7 @@ function launchMetals(
     revealOutputChannelOn: RevealOutputChannelOn.Never
   };
 
-  // TODO fix this any
-  // I know this any here is a bit gross, but I can't
-  // get any of the sendRequests to work or type check
-  // correctly without it. I notice that this was also
-  // done in some of the other coc-extensions, which
-  // game me the idea to try it this way, and it works
-  const client: any = new LanguageClient(
+  const client = new LanguageClient(
```

Comment:
see my comment above, this should be ok now.

Reply:
Awesome. Really happy to get this fixed.